client/src/components/Card/Card.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Card.css'
+import Icon from '../CardIcon'
+
+
+const Card = props => {
+	let cards = [
+		{
+			icon: <Icon name='heart' />,
+			title: 'Get Inspired',
+			p: 'Each design evokes classic couture while expressing a vision of modern style & sophistication.'
+		},
+		{
+			icon: <Icon name='calendar' />,
+			title: 'Book With Us',
+			p: 'Spend an hour with us to create a truly unique, personalized & flattering wedding dress.'
+		},
+		{
+			icon: <Icon name='dress' />,
+			title: 'Create Your Dream Dress',
+			p: 'Anything and everything is customizable to make you feel confident & beautiful.'
+		}
+	]
```

Comment:
This is all information that should be passed in as props - probably to create 3 instances of this `Card` component